src/main/resources/i18n/messages_fr.properties

```diff
@@ -37,6 +37,8 @@ addAnnouncement.start=Du :
 addAnnouncement.end=Au :
 addAnnouncement.save=Enregistrer
 addAnnouncement.charactersremaining=caract&egrave;res restants.
+addAnnouncement.attachments=Pi&egrave;ces jointes

```

Comment:
I am not sure, I was just following the standard put forth by the other French messages in the file, which also use the special references.